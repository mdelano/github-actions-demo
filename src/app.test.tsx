import React from 'react'
import { render } from '@testing-library/react'
import App from './app'

describe('app', () => {
  test('renders root app component correctly', () => {
    const { asFragment } = render(<App firstName="Yus" />)
    expect(asFragment()).toMatchSnapshot()
  })
})
