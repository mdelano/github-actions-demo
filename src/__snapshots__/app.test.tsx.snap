// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app renders root app component correctly 1`] = `
<DocumentFragment>
  <div>
    Welcome github actions!
  </div>
</DocumentFragment>
`;
